### PR TITLE
Update the command summary to our product features

### LINKED ISSUE #29: Update UG Command Summary

### PR DESCRIPTION
**Pull request by JayChenYJ**
Closes #29

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -183,10 +183,8 @@ _Details coming soon ..._
...
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List** | `list`
-**Help** | `help`
+**Add** | `add n/ITEM_NAME l/LOCATION q/QUANTITY [e/EXPIRY_DATE] [t/TAG]…​` <br> e.g., `add n/koko krunch l/fridge q/1 e/2021-05-27 t/amanda's favourite`
+**Delete** | `delete n/ITEM_NAME l/LOCATION`<br> e.g., `delete n/koko krunch l/fridge`
+**Update** | `update INDEX [n/ITEM_NAME] [l/LOCATION] [q/QUANTITY] [t/TAG]…​`<br> e.g.,`update 1 l/freezer q/2 `
+**List** | `list [l/LOCATION]`<br> e.g., `list l/fridge`
+**Search** | `search n/ITEM_NAME`<br> e.g, `search n/koko krunch`
```
Comment (Md-Fazil) [inline docs/UserGuide.md]: Should the command be find instead of search?

**Review by Md-Fazil — COMMENTED** [review]
Some nits and grits to fix before merging.

File: docs/UserGuide.md (hunk 1 above)
Comment (JayChenYJ) [reply docs/UserGuide.md]: Fixed :)

**Review by Md-Fazil — APPROVED** [review]
LGTM 👍

**Pull request merged**